docs/new_services.rst

```diff
@@ -0,0 +1,41 @@
+.. _topics-new_services:
+
+New Services
+============
+
+"Service" refers generally to any process, possibly long-running, which you
```

Comment:
More importantly, it's *multiple* processes. The expectation for a `Service` is that it might be a distributed system, so we by default expect multiple processes/nodes.